wordpress-mobile/WordPress-iOS
# Removes reblogging support from the Reader.

Closes #3852 

To test:
Open the Reader while signed into wpcom.
Look at Freshly Pressed (its a good test candidate).
Confirm the reblog button no longer shows in the cells, or in the post details. 

@jleandroperez Can I ping you for a review? I know how much you like deleting code ;)

Needs Review: @jleandroperez
